react/src/components/FieldDisplayEditable.tsx

```diff
@@ -87,8 +93,9 @@ function EnhancedTextField({
             textFieldProps.onChange = (e: TextFieldEvent) => {
                 onEdit(field.fieldName, e.target.value === "" ? null : e.target.value);
             };
+
             children = [
-                ...enums[getFieldInEnums(field.fieldName)]?.map((option: string) => (
+                enums[getFieldInEnums(field.fieldName)]?.map((option: string) => (
```

Comment:
Dropping the spread operator means we get an array with one element that is another array, are you sure? I don't recall why the `[...thing]` pattern was used here. `.map` does already return an array that isn't a reference to an existing one.